### PR TITLE
Stream and depends fixes.

### PR DESCRIPTION
**Pull request by alula**
very small changes

### CODE REVIEW

File: src/main/java/pl/mplauncher/launcher/MPLauncherApp.java [hunk 1]
```diff
@@ -35,14 +35,15 @@ public void start(Stage stage) throws Exception {
         Logger bootstrapLogger = (Logger) LogManager.getLogger("Bootstrap");
         Logger launcherLogger = (Logger) LogManager.getLogger("Launcher");
 
-        System.setOut(new PrintStream(new Log4j2OutputStream(bootstrapLogger, Level.INFO)));
```
Comment (kacperduras) [inline src/main/java/pl/mplauncher/launcher/MPLauncherApp.java:38]: why?

File: src/main/java/pl/mplauncher/launcher/MPLauncherApp.java [hunk 2]
```diff
@@ -35,14 +35,15 @@ public void start(Stage stage) throws Exception {
...
+        // logs should be printed only on one stream...
```
Comment (kacperduras) [inline src/main/java/pl/mplauncher/launcher/MPLauncherApp.java]: unnecessary comment.

File: src/main/java/pl/mplauncher/launcher/MPLauncherApp.java [hunk 3]
```diff
@@ -35,14 +35,15 @@ public void start(Stage stage) throws Exception {
...
+        System.setErr(new PrintStream(new Log4j2OutputStream(bootstrapLogger, Level.INFO)));
         System.setErr(new PrintStream(new Log4j2OutputStream(bootstrapLogger, Level.ERROR)));
 
         bootstrapLogger.log(Level.INFO, "MPLauncher v2.0.0-dev2.");
         bootstrapLogger.log(Level.INFO, "Our websites: https://mplauncher.pl / " +
                                                 "https://github.com/MPLauncher/");
-        bootstrapLogger.log(Level.WARN, "Copyright 2017 MPLauncher Team. Licensed under the Apache License, " +
-                                                "Version 2.0.");
+        bootstrapLogger.log(Level.INFO, "Copyright 2017 MPLauncher Team. Licensed under the Apache License, " +
+                                                "Version 2.0."); // why warn?
```
Comment (kacperduras) [inline src/main/java/pl/mplauncher/launcher/MPLauncherApp.java]: same.

**Review by kacperduras — CHANGES_REQUESTED** [review]

**Comment by alula** [comment]
@kacperduras done

**Comment by kacperduras** [comment]
@gabixdev 
```
System.setErr(new PrintStream(new Log4j2OutputStream(bootstrapLogger, Level.INFO)));
System.setErr(new PrintStream(new Log4j2OutputStream(bootstrapLogger, Level.ERROR)));
```

I don't think so.

**Comment by alula** [comment]
i have noticed that, now fixing xD

File: src/main/java/pl/mplauncher/launcher/MPLauncherApp.java [hunk 4]
```diff
@@ -35,15 +35,18 @@ public void start(Stage stage) throws Exception {
         Logger bootstrapLogger = (Logger) LogManager.getLogger("Bootstrap");
         Logger launcherLogger = (Logger) LogManager.getLogger("Launcher");
 
-        System.setErr(new PrintStream(new Log4j2OutputStream(bootstrapLogger, Level.INFO)));
-        System.setErr(new PrintStream(new Log4j2OutputStream(bootstrapLogger, Level.ERROR)));
+        System.setOut(new PrintStream(new Log4j2OutputStream(bootstrapLogger, Level.INFO, "[STDOUT] ")));
+        System.setErr(new PrintStream(new Log4j2OutputStream(bootstrapLogger, Level.INFO, "[STDERR] ")));
```
Comment (kacperduras) [inline src/main/java/pl/mplauncher/launcher/MPLauncherApp.java]: All prefixes should be configured by a .xml file.

**Review by kacperduras — CHANGES_REQUESTED** [review]

**Comment by kacperduras** [comment]
I close, it's a shame to talk.

**Pull request closed without merging**